src/test/resources/suite-testng1.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="SuitaTestNG" verbose="1" >
+    <test name="AltTest1" >
```

Comment:
You can remove entirely what's between `<test name="AltTest1"> ... </test>` and leave only the second `<test>`, the one for "DemoTests". Thanks!